addons/skin.pm3.hd/720p/Font.xml

```diff
@@ -1,181 +1,469 @@
 		</font>
 	 	<font>
 			<name>font35b</name>
-			<filename>LiberationSans-Regular.ttf</filename>
+			<filename>arial.ttf</filename>
 			<size>80</size>
 			<aspect>.8</aspect>
 		</font>
 	</fontset>
-	<fontset id="Arial" unicode="true">
+
+		<fontset id="Arial Mid" unicode="true">
 	 	<font>
 			<name>font10</name>
 			<filename>arial.ttf</filename>
-			<size>15</size>
+			<size>19</size>
 		</font>
 	 	<font>
 			<name>font11</name>
 			<filename>arial.ttf</filename>
-			<size>19</size>
+			<size>23</size>
 		</font>
 	 	<font>
 			<name>font12</name>
 			<filename>arial.ttf</filename>
-			<size>20</size>
+			<size>24</size>
 		</font>
 	 	<font>
 			<name>font12b</name>
 			<filename>arial.ttf</filename>
-			<size>20</size>
+			<size>24</size>
 			<aspect>.9</aspect>
 		</font>
 	 	<font>
 			<name>font13</name>
 			<filename>arial.ttf</filename>
-			<size>23</size>
+			<size>27</size>
 		</font>
 	 	<font>
 			<name>font18</name>
 			<filename>arial.ttf</filename>
-			<size>28</size>
+			<size>32</size>
 			<aspect>.8</aspect>
 		</font>
 	 	<font>
 			<name>font18b</name>
 			<filename>arial.ttf</filename>
-			<size>28</size>
+			<size>32</size>
 			<aspect>.8</aspect>
 		</font>
 	 	<font>
 			<name>font20</name>
 			<filename>arial.ttf</filename>
-			<size>25</size>
+			<size>29</size>
 		</font>
 	 	<font>
 			<name>font22</name>
 			<filename>arial.ttf</filename>
-			<size>28</size>
+			<size>32</size>
 		</font>
 	 	<font>
 			<name>font25</name>
 			<filename>arial.ttf</filename>
+			<size>39</size>
+		</font>
+	 	<font>
+			<name>font35b</name>
+			<filename>arial.ttf</filename>
+			<size>80</size>
+			<aspect>.8</aspect>
+		</font>
+	</fontset>
+	
+		<fontset id="Arial Big" unicode="true">
+	 	<font>
+			<name>font10</name>
+			<filename>arial.ttf</filename>
+			<size>22</size>
+		</font>
+	 	<font>
+			<name>font11</name>
+			<filename>arial.ttf</filename>
+			<size>29</size>
+		</font>
+	 	<font>
+			<name>font12</name>
+			<filename>arial.ttf</filename>
+			<size>30</size>
+		</font>
+	 	<font>
+			<name>font12b</name>
+			<filename>arial.ttf</filename>
+			<size>30</size>
+			<aspect>.9</aspect>
+		</font>
+	 	<font>
+			<name>font13</name>
+			<filename>arial.ttf</filename>
+			<size>33</size>
+		</font>
+	 	<font>
+			<name>font18</name>
+			<filename>arial.ttf</filename>
+			<size>38</size>
+			<aspect>.8</aspect>
+		</font>
+	 	<font>
+			<name>font18b</name>
+			<filename>arial.ttf</filename>
+			<size>38</size>
+			<aspect>.8</aspect>
+		</font>
+	 	<font>
+			<name>font20</name>
+			<filename>arial.ttf</filename>
 			<size>35</size>
 		</font>
 	 	<font>
+			<name>font22</name>
+			<filename>arial.ttf</filename>
+			<size>38</size>
+		</font>
+	 	<font>
+			<name>font25</name>
+			<filename>arial.ttf</filename>
+			<size>45</size>
+		</font>
+	 	<font>
 			<name>font35b</name>
 			<filename>arial.ttf</filename>
 			<size>80</size>
 			<aspect>.8</aspect>
 		</font>
 	</fontset>
-	<fontset id="Unicode" unicode="true">
```

Comment:
I added 2 section fonsets Arial Mid and Arial Big - for blind man like me - it is only increase letters size and put it in top of file. While GitHub desided i try to delete some lines - i don`t know ((((